src/ripple/rpc/handlers/AMMInfo.cpp

```diff
@@ -127,7 +127,7 @@ doAMMInfo(RPC::JsonContext& context)
     asset1Balance.setJson(ammResult[jss::amount]);
     asset2Balance.setJson(ammResult[jss::amount2]);
     lptAMMBalance.setJson(ammResult[jss::lp_token]);
-    ammResult[jss::trading_fee] = (*amm)[sfTradingFee];
+    ammResult[jss::trading_fee] = (*amm)[~sfTradingFee].value_or(0);
```

Comment:
This can safely be changed back to the original code `ammResult[jss::trading_fee] = (*amm)[sfTradingFee];`. If the field isn't set, it'll return 0.